modules/installation-configuration-parameters.adoc

```diff
@@ -597,6 +597,10 @@ accounts for the dramatically decreased machine performance.
 |The number of control plane machines to provision.
 |The only supported value is `3`, which is the default value.
 
+|`kmsKeyARN`
```

Comment:
Hi @xenolinux , From my understanding, `kmsKeyARN ` parameter should has 2 different yaml path `compute.platform.aws.rootVolume.kmsKeyARN` and `controlPlane.platform.aws.rootVolume.kmsKeyARN`. It’s better to add it to Optional AWS configuration parameters.

https://docs.openshift.com/container-platform/4.6/installing/installing_aws/installing-aws-customizations.html#installation-configuration-parameters-optional-aws_installing-aws-customizations

Reply:
Hi @yunjiang29 , The doc enchance is installer related, could you please help to take a look when you get a chance? If I missed something just correct me. Thanks!

Reply:
Addressed